### PR TITLE
Versioning guidelines

### PR DESCRIPTION
**Pull request by FrancescoCioria**

### CODE REVIEW

**Comment by FrancescoCioria** [comment]
@giogonzo @gabro

File: README.md [hunk 1]
```diff
@@ -24,3 +24,13 @@ Or from the package folder:
 NOTE: If you installed it as a GitHub dependency (`npm i --save buildo/react-component`) then import from `/src`:
 
 `import { FlexView } from 'buildo-react-components/src/flex';`
+
+### Versioning guidelines
+A new version of `b-r-c` is published every ~2 weeks with a github release containing a list of all the commits (mostly PR) merged on `master` since the previous one.
```
Comment (FrancescoCioria) [inline README.md]: maybe the list should contain only PRs as anything commited directly on master shouldn't be relevant to any project (like fixing an example or the readme)

Comment (gabro) [inline README.md]: merge commits are fine

File: README.md [hunk 2]
```diff
@@ -24,3 +24,13 @@ Or from the package folder:
...
+Each commit will marked as `Breaking`, `Fix`, `Feature`, `Docs`, `Build` (tests, examples, webpack...).
+example of a release:
+```
+Breaking: title of the PR (issue #123)
+Docs: title of the commit
+```
+To enforce this structure every commit merged on `master` should be already formatted that way. This means that before merging a PR the reviewer **must** format the merge commit.
```
Comment (giogonzo) [inline README.md]: nice 👍

**Pull request merged**